Dockerfile

```diff
@@ -21,22 +21,29 @@ RUN apk update && \
         proj-dev \
         make
 
-COPY requirements.txt .
-
-RUN pip install \
-    --prefix=/install \
-    --no-cache-dir \
-    --no-warn-script-location \
-    -r requirements.txt
+# Setup backend Python dependencies
+COPY pyproject.toml pdm.lock README.md ./
+RUN pip install --no-cache-dir --upgrade pip \
+    && pip install --no-cache-dir pdm==2.0.2 \
+    && pdm config python.use_venv false
```

Comment:
Do we also need `--pep582` ?

Reply:
Running that will set the PATH variable for the current shell session, but we need it to persist throughout the docker layers. I do that here https://github.com/hotosm/tasking-manager/blob/385d7c77e8c0eeafc750dbcc4bb508ac08b60331/Dockerfile#L36

If running tasking manager without docker, the user may have to run --pep582 first to recognise the installed dependencies dir `__pypackages__`